internal/namespaces/ultils.go

```diff
@@ -0,0 +1,38 @@
+package namespaces
+
+import (
+	"context"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func IsNamespaceInstalled(ctx context.Context, cs *kubernetes.Clientset, namespace string) (bool, error) {
```

Comment:
Personally, I would change the name to simply `Exists`, since
1. A namespace is not installed and
2. the package is already called namespaces